shell/components/nav/WindowManager/index.vue

```diff
@@ -293,6 +293,30 @@ export default {
 
     emitDraggable(event) {
       this.$emit('draggable', event);
+    },
+
+    resizeVertical(arrowUp) {
+      const resizeStep = 20;
+      const height = arrowUp ? this.height + resizeStep : this.height - resizeStep;
+
+      this.height = height;
+
+      this.setWmDimensions(height);
+    },
```

Comment:
It seems that I'm able to produce a scenario where the active terminal area gets clipped by dead space: 

![image](https://github.com/user-attachments/assets/8306dc27-50a5-4aab-aa57-c729a3118e56)

I'm not able to produce this scenario in `master`

Reply:
Should be fixed mate